tests/coafileTest.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import re
+import os
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.ConfigurationGathering import load_config_file
+
+
+class coafileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.filepath = '.coafile'
+        self.assertTrue(os.path.isfile(self.filepath),
```

Comment:
I don't see the point of this, plus you could have just used os.exists. Passing a logPrinter is fine for now until we get it fully removed.